stock_free_quantity/models/product_template.py

```diff
@@ -0,0 +1,35 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    def _compute_quantities_dict(self):
+        prod_available = super()._compute_quantities_dict()
+        for template in self:
+            free_qty = 0
+            for p in template.with_context(active_test=False).product_variant_ids:
+                free_qty += p.free_qty
```

Comment:
@pedrobaeza Can we replace to read_group? I don`t know how read_group works with computed fields.

Reply:
I don't get what do you mean. Are you talking about using `read_group` inside a compute method? That's possible.

Reply:
<pre>
data = self.env['product.product'].with_context(active_test=False).read_group([
  ('product_tmpl_id', 'in', self.ids),
], ['product_tmpl_id', 'free_qty'], ['product_tmpl_id']):
res = {
  rec['product_tmpl_id'][0]: rec['free_qty'] for rec in data
}
for template in self:
  prod_available[template.id]["free_qty"] = res.get(template.id, 0.0)
</pre>

Reply:
Is it improve perfomance or not?

Reply:
Yeah, indeed you get a performance improvement. It's the usual technique for doing compute fields efficient.

Reply:
Well, I gave it a try but it dit not work.
I don't really have time to investigate this but I guess it should not be blocking.
Won't have time to really check if there is a significant performance boost.

Reply:
I checked and saw that read_group doesn`t allow add non stored field in the field_list section.